docs/howto/how_to_run_perception_module_on_your_local_computer.md

```diff
@@ -37,7 +37,7 @@ docker commit CONTAINER_ID apolloauto/apollo:NEW_DOCKER_IMAGE_TAG
 
 6. Start the new docker image (in host) and get into docker:
 ```
-./docker/scripts/dev_start.sh NEW_DOCKER_IMAGE_TAG
+./docker/scripts/dev_start.sh -image NEW_DOCKER_IMAGE_TAG
```

Comment:
Warning: This option has been replaced by "-t" and "--tag", please use the new one.

Usage: dev_start.sh [options] ...
OPTIONS:
    -C                     Pull docker image from China mirror.
    -h, --help             Display this help and exit.
    -t, --tag <version>    Specify which version of a docker image to pull.
    -l, --local            Use local docker image.

Reply:
That update / option appears to be in the `master` branch only. 

How can we document this for the `demo_dev` branch?

Reply:
You can create a pull request for demo_dev branch. It should allow you to select your target branch when you create pull request.